### DISCUSSION TITLE
Projects beta worked, now shows page-not-found error

### DISCUSSION THREAD

**Discussion by jankrassnigg** [community/community#9441] [Projects and Issues]
I have used the projects beta for a while, but from one day to the next, it stopped working for me entirely.

When I go to the projects page, I see this:

![image](https://user-images.githubusercontent.com/6001174/147782684-cb5e1f1e-3c09-4565-b841-745f06d9c307.png)

But clicking "Add Project" just shows me a 404:

![image](https://user-images.githubusercontent.com/6001174/147782722-f88269d8-69eb-449f-ac9f-81c960ee75d9.png)

This is also the case for all links to beta projects. For example on issues I see links to existing beta projects, but clicking those links all end up on the 404 page.

I personally applied for the beta with my account, but I am using it on a shared organization repository (https://github.com/ezEngine). If I do the same on a repository that belongs only to me, it seems to work, so I guess this has something to do with the fact that I used in on another organization.

**Comment by Blancobrunob** [comment]
did you got an answer? im having the same issue

**Reply by jankrassnigg** [reply]
Nope. As you can see, I was simply ignored.

**Reply by 2percentsilk** [reply]
Hi there - it's possible that the repository had projects disabled on it by the owner or that they made them private and you no longer have access to them. I would check with the organization owner to see if this is the case.

**Reply by jankrassnigg** [reply]
I'm the owner and I didn't make anything private.
